src/execution_plan/execution_plan_build/build_projection_ops.c

```diff
@@ -155,6 +156,35 @@ static void _combine_projection_arrays(AR_ExpNode ***exps_ptr, AR_ExpNode **orde
 	*exps_ptr = project_exps;
 }
 
+// traverse expression operation checking if any variadic operand name found in the rax
+static bool _is_exp_use_aliases(AR_ExpNode *exp, rax *projected_aliases) {
```

Comment:
Throughout this file, use names like `_exp_uses_aliases` instead; this is a misuse of `is`.